buildSrc/src/main/kotlin/BuildInfo.kt

```diff
@@ -85,9 +87,14 @@ open class BuildInfo(project: Project) {
   val commitId: String by lazy {
     // only run command once per build invocation
     if (project === project.rootProject) {
-      Runtime.getRuntime()
-        .exec(arrayOf("git", "rev-parse", "--short", "HEAD"), arrayOf(), project.rootDir)
-        .inputStream.reader().readText().trim()
+      val process = ProcessBuilder()
+        .command("git", "rev-parse", "--short", "HEAD")
+        .directory(project.rootDir)
+        .start()
+      process.waitFor().also { exitCode ->
+        if (exitCode == -1) throw RuntimeException(process.errorStream.reader().readText())
+      }
```

Comment:
Tangential improvement: changed this because the failure was getting swallowed and I wasn't sure why the commit ID was not being populated correctly on Windows.